### DISCUSSION TITLE
Doesn’t check default.json in a renovate-config repository

### DISCUSSION THREAD

**Discussion by volschin** [renovatebot/pre-commit-hooks#2982] [Q&A] [answered]
I wanna use the pre-commit for my default.json but it isn’t recognized, at least not by default.
Would be nice to make this possible.

My usage example here:
https://github.com/volschin/renovate-config

**Comment by viceice** [comment, marked as answer]
I think you need to override the files regex 

https://github.com/renovatebot/pre-commit-hooks/blob/main/.pre-commit-hooks.yaml#L7
